### PR TITLE
Displays a popup if you're converted into a shade who needs to follow instructions

### LINKED ISSUE #68918: Shades have easily ommitable information about being slaved to someone
## Testmerges:
#68763: 'Adds the Mothroach' by Justice12354 at commit 2265106aba
#68599: 'Refactors SM Delaminations to be strategies' by vincentiusvin at commit 606d9b0fbb
#67902: 'Lemon does pain' by LemonInTheDark at commit faa2958c50

## Reproduction:

Get killed then look away from your screen or be otherwise distracted. Look back that you are now spectating (most likely) a chaplain, without ghost HUD.

The information about what you can/can't do (attack your master) is written in **bold standard size text** that could be easily lost in chat spam, **especially** when the (most likely) dead-player-now-shade is not looking at the game.

image

Getting slaved to someone is pretty important information that shouldn't get easily omitted.
Especially important for anyone getting sharded first time by chaplain. (Pretty sure cult shades get the antag info popup.)

### PR DESCRIPTION
**Pull request by Jacquerel**
## About The Pull Request

Fixes https://github.com/tgstation/tgstation/issues/68918
A week or so ago I soulstoned someone and they just spent the whole time yelling at people to try and get me killed until I got tired of it and chucked them into some lava. After that they actually noticed the text telling them they were supposed to have been my new spectral pal.
Then I looked at the issue list and noticed someone else had this problem.

Text is easy to miss, so I made it display a big popup instead.
It looks like this.
image
<details>
  <summary>Example with a longer name</summary>

image
</details>
It also gives you the usual antag button to re-display it, in case you want to check who your master is.

This window will display if:

- You are placed into a soulstone for the first time by someone who is not in the cult.
- You are placed into a soulstone by someone who wasn't the original person who soulstoned you (they become your new master).
- You were previoulsy a cult shade but your soulstone is exorcised by a chaplain.
- You are still a cult shade but are soulstoned by a miner with a soulstone, which makes you loyal both to the cult and the miner. I am not sure how conflicts are resolved here but this isn't a mechanical change and is how the "text printed into chat" method already worked so that's up to general antagonist priority policy.

## Why It's Good For The Game

If you've suddenly gained a restriction similar to an AI law, it should be as obvious as possible, especially if it's potentially to be the friend of someone who just killed you.

## Changelog

:cl:
qol: Newly created Shades will be told much more clearly that they're supposed to be bound to their master, if they have one.
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
Why am I getting a linter error on a file I didn't change 😔

**Comment by tralezab** [comment]
Perhaps this should just be an antag datum at the point where you're following orders that may or may not be antagonistic

**Comment by Jacquerel** [comment]
> Perhaps this should just be an antag datum at the point where you're following orders that may or may not be antagonistic

I would be happy to do this, the only reason I originally didn't was in case this intersects with policy in some way where "being an antagonist" is a more open role than "being a shade created by the chaplain" is supposed to be.
That would make this implementation easier though probably.

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by Jacquerel** [comment]
Alright it took me a while to untangle the various ways that people can use soulstones on shades and what it normally prints to those shades but I _think_ that the presence of this new datum will now match anything that would previously (and still is) be printed into the chat log.

File: code/modules/antagonists/shade/shade_minion.dm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		return
+	master_name = team.master.current?.real_name
+
+/**
+ * A team containing just the shade and master, so you can know who your master is.
+ */
+/datum/team/shade_pact
+	show_roundend_report = FALSE
+	/// The master of the pact
+	var/datum/mind/master
```
[the remark this answers is not in the document]
Comment (Jacquerel) [reply code/modules/antagonists/shade/shade_minion.dm]: I don't know why I did it that way when this way is much simpler.

**Comment by Jacquerel** [comment]
I am not too proud to say that I do not know how to satisfy the linter here.
The call stack seems to be upset that the Trigger proc in a different typepath than the one I am calling it on sleeps, even though that typepath cannot be involved.
Am I just not able to have ui_interact trigger inside of this proc due to the no sleep requirement in the chaplain rite? Do I need to refactor the xemo slime clothing change input to not sleep? What do I do about this?

**Pull request merged**